labs/2-perpetual-exchange/3-jlp-pool.md

```diff
@@ -35,6 +36,9 @@ Now, let's explore some of the risks associated with holding JLP:
 
 **Profit and Loss (P&L) Dynamics:** Traders' P&L from perpetual trading impacts the JLP pool. If a trader incurs a net positive P&L, the losses are sourced from the JLP pool to compensate the trader. Conversely, if a trader's P&L is net negative, the gains are deposited into the JLP pool for LP holders.
 
+JLP pool will lose in token value but not in USD value because the underlying token value in the pool appreciate in value as well. Read more about a reasearch from community member on how this work.
+https://skribr.io/app/article/exploring-jupiters-perpetual-futures-a-comprehensive-research-analysis/
```

Comment:
pull the graph to here and point to this article to ask the users to read more there.